gen3_util/projects/__init__.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from gen3.auth import Gen3Auth
+
+from gen3_util.config import ensure_auth, Config
+
+
+def get_user(config: Config = None, auth: Gen3Auth = None) -> dict:
+    """Fetch information about the user."""
+    if not auth:
+        assert config
+        auth = ensure_auth(config.gen3.refresh_file)
+    return auth.curl('/user/user').json()
+
+
+def get_buckets(config: Config = None, auth: Gen3Auth = None) -> dict:
+    """Fetch information about the buckets."""
+    if not auth:
+        assert config
+        auth = ensure_auth(config.gen3.refresh_file)
+
+    response = auth.curl('/user/data/buckets')
+
+    if response.status_code == 405:
+        logging.getLogger(__name__).warning(
+            "TODO /data/buckets response returned 405, "
+            "see https://cdis.slack.com/archives/CDDPLU1NU/p1683566639636949 "
+            "see quay.io/cdis/fence:feature_bucket_info_endpoint "
+        )
```

Comment:
No longer necessary? (as of only a couple hours ago haha -- endpoint added to latest master branch of fence)

Reply:
yes I get a 405 too at: https://aced-training.compbio.ohsu.edu/user/data/buckets with default fence image.